include/evmc/evmc.h

```diff
@@ -419,6 +419,8 @@ struct evmc_result
      *  freed with evmc_result::release().
      *
      *  This MAY be NULL.
+     *
+     * If output_size is NULL this pointer MAY NOT be dereferenced.
```

Comment:
```suggestion
     * If result::output_size is 0 this pointer MUST NOT be dereferenced.
```
and combine with the paragraph above.